AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -222,6 +222,12 @@ public class CardComponent: PresentableComponent,
             type: data.type
         )
         infoEvent.target = data.target
+        infoEvent.brand = data.brands?.first?.type.rawValue
+        if data.type == .selected {
```

Comment:
If type is anything but selected, we are adding the config data? Like `unfocus`, `validationError` for example. Am I seeing it wrong? :D
Maybe there is no tests for this check, if not let's also add it.

Reply:
I have changed the condition of the check and also, I have already tests for this :)